apps/ledger-live-mobile/src/components/RootNavigator/ManagerNavigator.tsx

```diff
@@ -59,6 +60,16 @@ export default function ManagerNavigator() {
         },
       }}
     >
+      <Stack.Screen
+        name={ScreenName.FirmwareUpdate}
+        component={FirmwareUpdateScreen}
+        options={{
+          gestureEnabled: false,
+          headerTitle: () => null,
+          headerLeft: () => null,
+          headerRight: () => <Button Icon={Icons.CloseMedium} />,
```

Comment:
Settings the header here works well with the error/device actions (we can always access the close button) ? Just asking, i think it's better to set the closing/back arrow in the react-navigation header

Though: does this button do anything ? I don't see any navigation back etc.

Reply:
It is only set here too to avoid "flickering" (i.e. a brief moment where there's a back button and no close button that gets replaced, if we set here, the change in the header done in the component is in visible to the user). That's also the reason why there's a button with no action here, it is replaced inside the component later.